modules/local_storage/commerce_stock_local.install

```diff
@@ -293,3 +301,17 @@ function commerce_stock_local_install() {
     ])
     ->execute();
 }
+
```

Comment:
Not strictly speaking, but since these aren't entity schemas that can be updated, doesn't hurt to provide the SQL update hook for anyone already using the module.